torch/_vmap_internals.py

```diff
@@ -248,11 +249,12 @@ def vmap(func: Callable, in_dims: in_dims_t = 0, out_dims: out_dims_t = 0) -> Ca
         sequences out of the box.
     """
     warnings.warn(
-        'Please use functorch.vmap instead of torch.vmap '
-        '(https://github.com/pytorch/functorch). '
-        'We\'ve moved development on torch.vmap over to functorch; '
-        'functorch\'s vmap has a multitude of significant performance and '
-        'functionality improvements.',
+        'torch.vmap is no longer being maintained. Please use '
```

Comment:
I would remove this terminology :P I think the old terminology of "we've moved development over to functorch" is better.

Reply:
Hmm, if you think the old terminology is better then I probably won't merge this PR then